### ISSUE TITLE
Issue watching already minified files

### ISSUE THREAD

**Issue by timshannon** [gobblejs/gobble#63]
Lets say I have a source folder with already minified  js files:

```
|--src
   |--js    
      |--ractive.min.js
|--gobblefile.js   
```

And your gobble file is setup to just copy the already minified file:

```
var gobble = require("gobble");
module.exports = gobble("src");
```

Then run `gobble watch out`

You'll get a ENOENT error looking for a .map file.

If you manually add the map file:

```
|--src
   |--js    
      |--ractive.min.js
      |--ractive.min.js.map
|--gobblefile.js   
```

You'll remove the error, but it'll continually invalidate the build over and over
```
tshannon@git:~/workspace/test$ gobble watch out
gobble: build completed in 5037ms. Listening for changes...

gobble:



build invalidated (2 files changed). restarting
gobble: build completed in 4509ms. Listening for changes...

gobble:



build invalidated (2 files changed). restarting
gobble: build completed in 4458ms. Listening for changes...

gobble:



build invalidated (2 files changed). restarting
gobble: build completed in 3683ms. Listening for changes...

gobble:



build invalidated (2 files changed). restarting
gobble: build completed in 4205ms. Listening for changes...

gobble:
```

**Comment by Rich-Harris** [comment]
This was happening because gobble was writing the 'flattened' sourcemaps to the final directory of the build, rather than creating a separate one. In this case, the final directory is also the source directory... so flattening the sourcemap triggered an unnecessary rebuild. Is fixed in 0.9.3. Thanks for raising it

**Comment by timshannon** [comment]
Working great for me.

Thanks,
